requirements.txt

```diff
@@ -0,0 +1,3 @@
+BeautifulSoup==3.2.1
+lxml==3.4.4
+xbundle==0.1.0
```

Comment:
That's probably a better call for you to make than me. I'd rather see xbundle hit PyPi. However, I'm in the middle of working on xbundle (pylint), so if it's a PITA for someone to deploy to PyPi then maybe wait, because they're going to have to do it again in another day or two.